source/fundamentals/monitoring.txt

```diff
@@ -48,24 +48,24 @@ application by the driver:
      - Description
 
    * - ``serverOpening``
-     - Created when a connection to an instance is established.
+     - Created when a connection to an instance opens.
 
    * - ``serverClosed``
-     - Created when a connection to an instance is closed.
+     - Created when a connection to an instance closes.
 
    * - ``serverDescriptionChanged``
      - Created when an instance state changes (such as from secondary to
        primary).
 
    * - ``topologyOpening``
-     - Created before any instance connections are attempted.
+     - Created prior to attempting a connection to an instance.
```

Comment:
```suggestion
     - Created prior to establishing a connection to an instance.
```

Reply:
I think the nuance communicated with "attempting" is that it could fail whereas establishing implies that it was successful. I could be more specific and use "attempting to establish a connection" if you think "attempting a connection" sounds awkward. Let me know what you think. @nathan-contino-mongo

Reply:
I do think that "attempting to establish a connection" would sound better, but that's also a good chunk of words, so... "attempting" seems fine to me. Good point!